lib/dynamostorage.js

```diff
@@ -17,35 +16,26 @@ module.exports = function() {
 class DynamoStorage {
   constructor(link) {
     const environment = link.chain.config;
-    const dynamodb = new AWS.DynamoDB(_.merge({
+    this.client = new AWS.DynamoDB.DocumentClient(_.merge({
```

Comment:
turns out this npm package is deprecated and recommends we just use the document client from the AWS Sdk